bindata/etcd/pod.yaml

```diff
@@ -48,62 +48,65 @@ spec:
         #!/bin/sh
         set -euo pipefail
 
-        ETCDCTL="etcdctl --cacert=/etc/kubernetes/static-pod-resources/configmaps/etcd-serving-ca/ca-bundle.crt \
-                           --cert=/etc/kubernetes/static-pod-resources/secrets/etcd-all-peer/etcd-peer-NODE_NAME.crt \
-                           --key=/etc/kubernetes/static-pod-resources/secrets/etcd-all-peer/etcd-peer-NODE_NAME.key \
-                           --endpoints=${ALL_ETCD_ENDPOINTS}"
-        ${ETCDCTL} member list
-
-        echo "waiting for member ${NODE_NODE_ENVVAR_NAME_ETCD_DNS_NAME}..."
-        COUNT=30
-        while [ $COUNT -gt 0 ]; do
-          echo "current member list is..."
+        if [[ ! -d /var/lib/etcd/member/ ]]; then
```

Comment:
This is the only check added in this PR. Rest of the code is identical (but tabbed under if condition).